Library/Formula/hunspell.rb

```diff
@@ -35,7 +44,7 @@ def caveats; <<-EOS.undent
 +++ src/tools/hunspell.cxx	2013-08-02 18:20:27.000000000 +0200
 @@ -28,7 +28,7 @@
  #ifdef WIN32
- 
```

Comment:
Please try not to tweak the whitespacing in patches; doing so has the tendency to pass the CI and then break for users.